### ISSUE TITLE
Posix mode breaks cncli.sh script on Ubuntu 20.04

### ISSUE THREAD

**Issue by adavault** [cardano-community/guild-operators#1814]
**Describe the bug**
This commit breaks cncli.sh
https://github.com/cardano-community/guild-operators/commit/14c428132596c26e638c00dae0925a5992b7e436

**Additional context**

```
./cncli.sh: line 337: syntax error near unexpected token `<'
./cncli.sh: line 337: `      done < <(jq -c '.assignedSlots[]' <<< "${cncli_leaderlog}" 2>/dev/null)'

```
changing line 971 in env file to `set +o posix` fixes...

**Comment by marco-martins** [comment]
Also with Ubuntu 20.04.6 LTS I'm getting the following error when I run the cntools script, I think it is related. 

`opt/cardano/cnode/scripts/cntools.library: line 453: syntax error near unexpected token `<'
/opt/cardano/cnode/scripts/cntools.library: line 453: `  done < <(find "${1}" -mindepth 1 -maxdepth 1 -type d -print0 | sort -z)'`

**Comment by rdlrt** [comment]
The posix mode currently is in place due to a bug on bash for printing something like `printf "%.2f" 1` . It works fine on lower bash releases but not on latest ones (the fix for bash is merged upstream but not yet part of bash packaging - can see details [here](https://lists.gnu.org/archive/html/bug-bash/2024-08/msg00087.html) ).

Having said that, POSIX mode of bash on older version OS starts having different issues - one of which is the issue with process substitution  entined above

**Comment by rdlrt** [comment]
was fixed in 3b839bc
